### PR TITLE
Add scroll-margin-top to account for sticky header

### LINKED ISSUE #16263: Review comments are covered by sticky header when opening direct link
- Gitea version (or commit ref): 1.15
- Can you reproduce the bug at https://try.gitea.io:
  - [X] Yes https://try.gitea.io/jpraet/test/pulls/9/files#issuecomment-76046 

## Description

When navigating directly to a specific review comment link, like https://try.gitea.io/jpraet/test/pulls/9/files#issuecomment-76046, the review comment is covered by the sticky header.

## Screenshots

afbeelding

expected:

afbeelding

### PR DESCRIPTION
**Pull request by jpraet**
When navigating directly to a specific review comment link, like https://try.gitea.io/jpraet/test/pulls/9/files#issuecomment-76046, the review comment is covered by the sticky header.

This PR adds a scroll margin to account for this header.

Fixes #16263

### CODE REVIEW

File: web_src/less/_review.less [hunk 1]
```diff
@@ -173,3 +173,15 @@ a.blob-excerpt:hover {
...
 }
+
+.pull.files.diff [id] {
+  scroll-margin-top: 99px;
+
+  @media @mediaMd {
+    scroll-margin-top: 130px;
+  }
+
+  @media @mediaSm {
```
Comment (noerw) [inline web_src/less/_review.less]: You could ~~add `@mediaMdAndDown` to `variables.less` and~~ merge these blocks

**Review by noerw — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

File: web_src/less/_review.less (hunk 1 above)
Comment (zeripath) [reply web_src/less/_review.less]: There already is a @mediaMdAndDown...

File: web_src/less/_review.less [hunk 2]
```diff
@@ -173,3 +173,15 @@ a.blob-excerpt:hover {
 .review-box > .segment {
   border: none !important;
 }
+
+.pull.files.diff [id] {
+  scroll-margin-top: 99px;
+
+  @media @mediaMd {
```
Comment (zeripath) [inline web_src/less/_review.less]:
```suggestion
  @media @mediaMdAndDown {
```

File: web_src/less/_review.less [hunk 3]
```diff
@@ -173,3 +173,15 @@ a.blob-excerpt:hover {
...
+    scroll-margin-top: 130px;
+  }
+
+  @media @mediaSm {
+    scroll-margin-top: 130px;
+  }
```
Comment (zeripath) [inline web_src/less/_review.less]:
```suggestion
```

**Pull request merged**